src/main/resources/db/changelog/ddl/0020_alter_anonymous_elector_document.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="vishal.gupta@valtech.com" id="0020_EIP-4669_anonymous_elector_document add delivery info column" context="ddl">
+
+        <addColumn tableName="anonymous_elector_document">
+            <column name="delivery_id" type="uuid" afterColumn="request_date_time">
+                <constraints nullable="true"
```

Comment:
Note: It's a nullable column and we are not adding any new migration scripts for this.

Reply:
It needs to be nullable as we are removing the delivery related info after 15 months (before the data that's on the printed document itself, which is removed after 10 years).